src/ibpm.h

```diff
@@ -26,6 +26,7 @@
 #include "BoundaryVector.h"
 #include "BaseFlow.h"
 #include "State.h"
+#include "StateVector.h"
```

Comment:
Does this need to be included in this file?  I don't see why it does, but maybe I am missing something.

Reply:
I included it because I thought StateVector would be a useful addition to the library that is created.